presto-jdbc/src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -591,4 +622,26 @@ public Statement getStatement()
             return statement;
         }
     }
+
+    private boolean serverSupportsVariablePrecisionTimestamp()
+    {
+        return getNumericPrestoServerVersion() >= 335;
+    }
+
+    private boolean serverSupportsVariablePrecisionTimestampWithTimeZone()
+    {
+        return getNumericPrestoServerVersion() >= 337;
+    }
+
+    private int getNumericPrestoServerVersion()
+    {
+        try {
+            return Integer.parseInt(prestoServerVersion.get());
+        }
+        catch (NumberFormatException e) {
+            log.info("Could not parse Presto version '%s' obtained from server; assuming server is recent");
```

Comment:
recent -> latest

(it's the latest, aka the current version, not "some relatively recent")